_posts/2020-04-12-final-presentations-youre-all-invited.md

```diff
@@ -12,19 +12,19 @@ photos:
   license: Public domain
 ---
 
-It's been a long and exciting journey, from applications opening in October 2019, [announcing selected applicants in early January]({% link _posts/2020-01-13-ols1-announcement.md %}), and kicking off [the program's first cohort call on the 29th of January]({% link _ols-1/week02.md %}). Since then we've had an amazing range of guest speakers from around the world, sharing their expertise on topics in [the OLS syllabus]({% link ols-1.md %}), as well as a variety of breakout and discussion sessions, interspersed with mentor calls every other week. We wrapped up our learning calls with [a cohort call on self care, ally skills, and careers]({% link _ols-1/week11.md %}).  
+It's been a long and exciting journey, from applications opening in October 2019, [announcing selected applicants in early January]({% link _posts/2020-01-13-ols1-announcement.md %}), and kicking off [the program's first cohort call on the 29th of January](../openseeds/ols-1/week02.md). Since then we've had an amazing range of guest speakers from around the world, sharing their expertise on topics in [the OLS syllabus](../openseeds/ols-1.md), as well as a variety of breakout and discussion sessions, interspersed with mentor calls every other week. We wrapped up our learning calls with [a cohort call on self care, ally skills, and careers](../openseeds/ols-1/week11.md).
```

Comment:
I would keep all `{% link ... %}` but change the link to the new file

Reply:
👋 Hi @bebatut.
Writing the path this way results in errors...as Jekyll does not recognise the path. I am not sure why...

```
[mentors]({% link /openseeds/ols-2/index.md#Mentors %})
```

The error:
```
/var/lib/gems/3.1.0/gems/jekyll-3.9.2/lib/jekyll/tags/link.rb:27:in `render': Could not find document 'openseeds/ols-2/index.md#experts' in tag 'link'. (ArgumentError)
Make sure the document exists and the path is correct.
```

Reply:
A link like this toward an anchor (e.g. `#Mentors`) does not work but without anchor (e.g. `{% link /openseeds/ols-2/index.md %}`) should work. 
When you have an anchor, you still need to the path

Reply:
I had done this previously as well and it did not work either. On the right, is the change and the output.

![Screenshot (560)](https://github.com/open-life-science/open-life-science.github.io/assets/105166953/ee7de6bb-a922-42c4-9d96-c59ed31374b2)

Reply:
When I meant the path, it was without `{% link ... %}`, like you did before

Reply:
Running to your Slack DM, please. 🙏